### PR TITLE
Challenges 36-40

### PR DESCRIPTION
**Pull request by shadowc**
This stuff is deep into turbo's guts so very few new concepts to really capture here.

### CODE REVIEW

File: challenges-36-plus.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          response, such as `body`, `headers` `status`, `redirected`
+          and more.
+        b: |
+          `location`: Is an object that contains the information about the
+          location where the response came from.
+        c: |
+          `language`: Is a string containing the language in which our content
+          was retrieved.
+        d: |
+          `succeded`: Is a boolean that determines weather the fetch was
```
Comment (weaverryan) [inline challenges-36-plus.yml:23]: Is this the right key? It looks like a typo - maybe `succeeded` - so we should check

File: challenges-36-plus.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          successful.
+      correct: c
```
Comment (weaverryan) [inline challenges-36-plus.yml:25]: Cool way to advertise some nice things on this :)

File: challenges-36-plus.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        b: |
+          Because we need to make sure that after our form submits,
+          the pages will load from scratch so that they show the
+          reflected changes (Turbo does not do this on its own)
+        c: |
+          Because it's always a good practice to clear the Turbo
+          cache before we call `Turbo.visit()` so that we always
+          get a fresh page.
+        d: |
+          Because Turbo has a bug when some times it doesn't properly
```
Comment (weaverryan) [inline challenges-36-plus.yml:51]:
```suggestion
          Because Turbo has a bug when sometimes it doesn't properly
```

**Review by weaverryan — COMMENTED** [review]
Minor notes!

File: challenges-36-plus.yml (hunk 1 above)
Comment (shadowc) [reply challenges-36-plus.yml:23]: I do have a typo there! I'll doublecheck

**Pull request merged**